src/lib/serviceRoutinesV2/postEntities.cpp

```diff
@@ -34,6 +34,12 @@
 #include "serviceRoutines/postUpdateContext.h"
 
 
+static bool legalEntityLength(Entity* eP, const std::string &servicePath)
+{
+  const int constant = 10;
```

Comment:
But for modern C and C++ the recommendation is use constants instead of #define maybe create a global file scoped constant? something like:
statict const int STRUCTURAL_OVERHEAD_BSON_ID = 10;